google-cloud-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITStorageHmacKeySnippets.java

```diff
@@ -0,0 +1,107 @@
+package com.google.cloud.examples.storage.snippets;
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.HmacKey;
+import com.google.cloud.storage.HmacKey.HmacKeyMetadata;
+import com.google.cloud.storage.HmacKey.HmacKeyState;
+import com.google.cloud.storage.ServiceAccount;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.testing.RemoteStorageHelper;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class ITStorageHmacKeySnippets {
```

Comment:
Is this a test? The class name should probably end in Test for clarity

Reply:
An integration test. I followed the other files in the same directory where they all used the `IT` prefix.